### DISCUSSION TITLE
custom score distributuions in mp lobbies

### DISCUSSION THREAD

**Discussion by sergcam** [ppy/osu#14349] [Ideas]
basically be able change the weight of accuracy vs combo in scores for a multiplayer lobby. i think this would be cool for nontraditional tourneys where accuracy might matter more than normal e.g. speed cup, so that you can increase the weight of accuracy without having to completely ignore combo. so for example in something like speed cup you could do like 80/20 acc/combo or im also playing in a low sr open rank tourney where something like 50/50 or 60/40 might make more sense. i think it would be an interesting way to add variety to tourneys

**Comment by peppy** [comment]
Definitely something we want to see happen.
